src/pages/proposal/create/MajoritySelection.tsx

```
@@ -0,0 +1,139 @@
+import React from 'react';
+import {
+  Box,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Slider,
+  Typography,
+  useTheme,
+} from '@mui/material';
+
+const MajoritySelection = ({ type, value, onChange }) => {
```

ESLint

[error] 'type' is missing in props validation

(react/prop-types)